### DISCUSSION TITLE
TuringCourses/political/military_theory/

### DISCUSSION THREAD

**Discussion by giscus** [ZJU-Turing/TuringCourses#167] [General]
# TuringCourses/political/military_theory/

图灵班学习指南

https://zju-turing.github.io/TuringCourses/political/military_theory/

**Comment by 626lt** [comment]
程春老师其实是会点名的，现在基本上都是雷达点名，一般都会提前通知并且在那次课进行开卷小测，小测是三次中取两次高的计分，而且很多都可以在ppt里面搜索到。而且根据最后的得分来看，区分度基本上都在期末的统一考试里面，程春老师的平时分只要你认真对待就会给到接近满分的分数。
